### PR TITLE
test: Fix shard evacuation test

### PR DESCRIPTION
**Pull request by acid-ant**
Close #2179

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Changelog for FrostFS Node
 ### Added
 ### Changed
 ### Fixed
+- Test `TestEvacuateNetwork/multiple_shards,_evacuate_many` (#2179)
```
Comment (fyrchik) [inline CHANGELOG.md:9]: I don't think we should mention tests in the CHANGELOG.

File: pkg/local_object_storage/engine/evacuate_test.go [hunk 2]
```diff
@@ -173,6 +184,12 @@ func TestEvacuateNetwork(t *testing.T) {
 		require.NoError(t, e.shards[ids[0].String()].SetMode(mode.ReadOnly))
 		require.NoError(t, e.shards[ids[1].String()].SetMode(mode.ReadOnly))
 
+		for _, shid := range ids {
+			objs, _ := e.shards[shid.String()].List()
+			fmt.Printf("Objs count on shard %s: %d\n", shid, len(objs.AddressList()))
```
Comment (fyrchik) [inline pkg/local_object_storage/engine/evacuate_test.go:189]: `t.Logf` is better for such purposes.

File: pkg/local_object_storage/engine/evacuate_test.go [hunk 3]
```diff
@@ -210,6 +227,20 @@ func TestEvacuateNetwork(t *testing.T) {
 		prm.handler = acceptOneOf(objects, totalCount-1)
 
 		res, err := e.Evacuate(prm)
+		if !errors.Is(err, errReplication) {
+			fmt.Println("!!! Connect and check!!!!!")
```
Comment (fyrchik) [inline pkg/local_object_storage/engine/evacuate_test.go:231]: What is this for?

File: pkg/local_object_storage/engine/evacuate_test.go (hunk 2 above)
Comment (acid-ant) [reply pkg/local_object_storage/engine/evacuate_test.go]: Removed as redundant.

File: pkg/local_object_storage/engine/evacuate_test.go (hunk 3 above)
Comment (acid-ant) [reply pkg/local_object_storage/engine/evacuate_test.go]: Sorry, removed. Where was my mind when I push this?)

File: CHANGELOG.md (hunk 1 above)
Comment (acid-ant) [reply CHANGELOG.md]: Removed.

**Pull request merged**